src/components/Common/FilePreviewDialog.tsx

```
@@ -395,6 +550,113 @@ const FilePreviewDialog = (props: FilePreviewProps) => {
+                        disabled={button[3] as boolean}
+                      >
+                        {button[1] && (
+                          <CareIcon
+                            icon={button[1] as IconName}
+                            className="mr-2 text-lg"
+                          />
+                        )}
+                        {button[0] as string}
+                      </Button>
+                    ))}
+                  </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)